applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml

```diff
@@ -18,7 +20,7 @@ ocil_clause: 'the kubelet cannot rotate server certificate'
 
 ocil: |-
     Run the following command on the kubelet node(s):
-    <pre>$ sudo grep -B1 RotateKubeletServerCertificate /etc/origin/node/node-config.yaml</pre>
+    <pre>$ sudo grep RotateKubeletServerCertificate /etc/origin/node/node-config.yaml</pre>
```

Comment:
For consistency with ``description`` and the rule name, should this be client or server?

Reply:
Mmmmm.... What would be the best way of handling multiple configs in one line? If you look at what we have traditionally done with the `pam` settings like `pam_faillock`, we show all options.
Having `....,OPTION,....` might be more confusing. Open to ideas.

Reply:
rule name updated to match the client rule name.